components/showcard.tsx

```diff
@@ -93,6 +93,28 @@ interface DescriptionsProps {
 }
 
 const Descriptions = ({ show, isExpanded }: DescriptionsProps) => {
+  // Add hyperlinks to words that start with https://
+  const linkedDescription = useMemo(() => {
+    if (!show.description) return null;
+    const words = show.description.split(/(\s+)/);
+    return words.map((word, idx) => {
+      if (word.startsWith('https://')) {
+        return (
+          <a
+            key={idx}
+            href={word}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="underline hover:text-radio-accent"
+          >
+            {word}
+          </a>
+        );
+      }
+      return <span key={idx}>{word}</span>;
```

Comment:
Ah korjasitkin edellisen kommentin, mut there's more 😁 
```suggestion
      return <Fragment key={idx}>{word}</Fragment>;
```